pkg/storage/fs/ocis/node.go

```diff
@@ -387,6 +392,12 @@ func (n *Node) AsResourceInfo(ctx context.Context) (ri *provider.ResourceInfo, e
 		log.Error().Err(err).Interface("node", n).Msg("could not list attributes")
 	}
 
+	if common.FindString(mdKeys, _ShareTypesKey) != -1 {
+		if n.hasUserShares(ctx) {
+			ri.ArbitraryMetadata.Metadata[_ShareTypesKey] = "0"
```

Comment:
I was confused by the `"0"` and needed to ask. For the record, it would be really cool to have constants for the share types.

Reply:
Yes, I agree. :+1: